esp8266/modules/webrepl.py

```diff
@@ -75,3 +70,51 @@ def start_foreground(port=8266):
     stop()
     s = setup_conn(port, None)
     accept_conn(s)
+
+def _sanitize_password(password):
+    if not password:
+            raise ValueError("Password is required")
+    # Check length.
+    if not 4 <= len(password) <= 8:
+        raise ValueError("Password has to be between 4 and 8 characters long")
+    # Only allow printable ASCII characters.
+    for c in password:
+        if not 32 <= ord(c) <= 126:
+            raise ValueError("Invalid character in the password")
+    # Escape backslashes and quotes.
+    password = password.replace('\\', '\\\\').replace("'", "\\'")
+    return password
+
+def _change_daemon(enable, password=''):
+    LINE_START = "import webrepl; webrepl.start(password='"
```

Comment:
Let me return your question: What if a user splits that into 2 lines? What if one puts 2 spaced after ";"? 3? 5? what if they use `"` as a string delimiter? I hope it's now clear that it's a dumb question. "If a user wants to fool the script, they can - but they fool themselves, not script." However, you really try to "parse the code", while I tried to minimize need for that as much as possible. Thus, in real-world conditions your script would fail with a higher probability, while violating the config-vs-code separation best practice.